sale_procurement_group_by_requested_date/__openerp__.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# Copyright 2017 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Group procurements by requested date",
+    "summary": "Groups pickings based on requested date of order line",
+    "version": "9.0.0.0.1",
+    "category": "Sales Management",
+    "website": "http://www.eficent.com",
+    "author": "Eficent Business and IT Consulting Services S.L., "
```

Comment:
with OCA I mean "Odoo Community Association (OCA)" which is always the last author in the list.